src/Traits/Users/ReferenceNumberTrait.php

```diff
@@ -8,6 +8,8 @@
 trait ReferenceNumberTrait
 {
     /**
+     * @Groups({"create"})
```

Comment:
I dont believe we allow reference numbers to be chosen?

Reply:
I think the controller code in payments lets us, but will check

Reply:
yeah, that's wrong, will remove.

Reply:
💥